plugins/modules/web_infrastructure/jenkins_build.py

```diff
@@ -176,6 +199,13 @@ def present_build(self):
             self.module.fail_json(msg='Unable to create build for %s: %s' % (self.jenkins_url, to_native(e)),
                                   exception=traceback.format_exc())
 
+    def stopped_build(self):
+        try:
```

Comment:
Is it not possible to do idempotency checking? (I also saw it isn't done for `state=present` and `state=absent` either, which looks a bit ... strange to me.)

Reply:
I will check the expected result of stop_build in python-jenkins in repeating requests and add the idempotent test case for this function maybe tomorrow. (I believe stop a build is idempotent as I remember it always return None for any existing build)
As for create a build or delete a build, I think the return result will be different for every requests, I will try to analyse them but maybe not implemented in this PR.

Reply:
The idempotent test case for the stop build function is added in the latest commit. As for create or delete a build, they would not be implemented in this PR as more investigation is needed.

Reply:
Yes, that's definitely not something for this PR :)